packages/amplify-graphql-model-transformer/rds-lambda/package.json

```diff
@@ -6,15 +6,17 @@
     "typescript": "^4.8.4"
   },
   "dependencies": {
+    "@aws-sdk/client-ssm": "^3.264.0",
     "babel-jest": "^29.1.2",
     "jest": "^29.1.2",
     "knex": "^2.3.0",
+    "mysql": "^2.18.1",
```

Comment:
We used `mysql2` in our POC, it has few advantages over the other version.
https://www.npmjs.com/package/mysql2

Reply:
done